source/_patterns/01-molecules/components/personalised-cover-download.mustache

```diff
@@ -25,12 +25,21 @@
       {{/text}}
     </div>
     <div class="personalised-cover-thumbnail">
-      <picture>
-        <source srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/cover-preview.jpg{{/assetRewrite}}" type="image/svg+xml">
-        <img srcset="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/cover-preview.jpg{{/assetRewrite}} 276w"
-          src="{{#assetRewrite}}{{assetsPath}}/img/patterns/molecules/cover-preview.jpg{{/assetRewrite}}" sizes="138px"
-          alt="" class="personalised-cover-download__icon">
-      </picture>
+      {{#image}}
+        <picture class="profile-snippet__picture">
```

Comment:
This class name could possibly be more specific to personalised covers, or removed?

Reply:
Testing it too, it isn't suitable because then it has `max-width: 4.5rem;` and the image appears too small, so remove or new class name? Thanks!